Deploy/bash/initialize-greenfield-environment.sh

```diff
@@ -435,10 +435,10 @@ configure_greenfield_environment ()
   echo "Initialize and validate terraform for the ${CMS_ENV_GE} environment..."
   echo "******************************************************************************"
 
-  cd "${START_DIR}/.."
-  cd terraform/environments/$CMS_ENV_GE
+  cd "${START_DIR}/.." || exit
+  cd terraform/environments/"${CMS_ENV_GE}" || exit
```

Comment:
Is there any log message when you exit like this?

Reply:
Yes, if "cd" fails, it logs the error message and prevents the script from continuing.
See here:
https://github.com/koalaman/shellcheck/wiki/SC2164